clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -210,7 +210,7 @@ public class AzureComputeClient extends AzureBaseClient {
   Collection<AzureInstance> getServerGroupInstances(String resourceGroupName, String serverGroupName) {
     def instances = new ArrayList<AzureInstance>()
 
-    executeOp({scaleSetVMOps.list(resourceGroupName, serverGroupName, null, null, "instanceView")}).body.each {
+    executeOp({scaleSetVMOps.list(resourceGroupName, serverGroupName, null, null, "instanceView")})?.body?.each {
```

Comment:
This is one of the cases where we blew up trying to retrieve the resources associated with the server group in the server group caching agent.